addons/io_hubs_addon/components/definitions/reflection_probe.py

```diff
@@ -803,13 +870,30 @@ def draw(self, context, layout, panel):
     def gather(self, export_settings, object):
         from ...io.utils import gather_texture
         return {
-            "size": object.data.influence_distance,
+            "size": self.influence_distance,
+            "clipStart": self.clipStart,
+            "clipEnd": self.clipEnd,
             "envMapTexture": {
                 "__mhc_link_type": "texture",
                 "index": gather_texture(self.envMapTexture, export_settings)
             }
         }
 
+    def migrate(self, migration_type, panel_type, instance_version, host, migration_report, ob=None):
+        migration_occurred = False
+
+        if self.type == 'LIGHT_PROBE' and instance_version < (1, 0, 1):
```

Comment:
This line errors out.  I think you probably wanted `host.type`?

Reply:
Updated.